packages/react-error-overlay/package.json

```diff
@@ -40,7 +40,7 @@
     "babel-eslint": "10.0.3",
     "babel-jest": "^24.9.0",
     "babel-loader": "8.0.6",
-    "babel-preset-react-app": "^9.0.2",
+    "babel-preset-react-app": "9.0.0",
```

Comment:
This is due to a bug in upstream create-react-app here: `https://github.com/facebook/create-react-app/issues/8680`

Reply:
That bug is now fixed btw ^